chromium_src/components/sync/base/features.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/components/sync/base/features.cc"
+
+#include "base/feature_override.h"
+
+namespace syncer {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kSyncEnableHistoryDataType, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
This would be just a workaround right?
I think we need to investigate in HistoryDataType because I can foresee typed url data type will go away sooner or later. Please open an issue for that.

Reply:
yes this is a workaround to fix CR117 regression.

Proper solution is on the way, some investigations were done at https://github.com/brave/brave-core/pull/17216. 

Created issue: https://github.com/brave/brave-browser/issues/32876